website/docs/compute/local-deployment/minikube.md

```diff
@@ -0,0 +1,225 @@
+
+### 4. Generate your Compute manifests
+
+Next we’ll be using subo to generate our Kubernetes manifest files!
+
+```bash
+subo compute deploy core --dryrun
+```
+
+You will be asked for a domain. Please make sure to enter your domain from ngrok.
+
+These will now live in the `.suborbital/` folder.
+
+### 5. Disable TLS checks in the Compute environment
+
+Open up `.suborbital/scc-controlplane-deployment.yaml` in your editor of choice, and make the following changes.
+
+We are disabling the built in TLS certificate provisioning, as ngrok already takes care of this for us.
+
+Under the Builder Container:
+
+```yaml
+- name: builder
+          image: suborbital/scc-builder:v0.3.1
+          command: ["builder"]
+
+          ports:
+            - containerPort: 8080
+            - containerPort: 8443
+
+          env:
+            - name: SCC_DOMAIN
+              value: "<YOUR_NGROK_DOMAIN>"
+
+            - name: SCC_TLS_PORT
+              value: "8443"
+
+            - name: SCC_LOG_LEVEL
+              value: "info"
+
+            - name: SCC_CONTROL_PLANE
+              value: "scc-controlplane-service:8081"
+
+          volumeMounts:
+            - mountPath: "/home/scn"
+              name: controlplane-storage
+```
+
+Delete the following line:
+
+`delete containerPort: 8443`
+
+Delete the following key-value pair:
+
+```yaml
+- name: SCC_DOMAIN
+  value: "<YOUR_NGROK_DOMAIN>"
+```
+
+Replacing the following key-value pair:
+
+```yaml
+- name: SCC_TLS_PORT
+  value: "8443"
+```
+
+With the following:
+
+```yaml
+name: SCC_HTTP_PORT
+value: "8080"
+```
+
+Under the `scc-builder-service`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: suborbital
+  name: scc-builder-service
+spec:
+  selector:
+    app: scc-controlplane
+  ports:
+    - protocol: TCP
+      name: challenge
+      port: 80
+      targetPort: 8080
+    - protocol: TCP
+      name: https
+      port: 443
+      targetPort: 8443
+  type: LoadBalancer
+```
+
+Our builder service no longer needs to expose HTTPS ports as ngrok will forward both HTTP and HTTPS traffic to port 80.
+
+Remove the following lines:
+
+```yaml
+- protocol: TCP
+  name: https
+  port: 443
+  targetPort: 8443
+```
+
+### 6. Deploy to your cluster
+
+Run the following `subo` command to deploy Compute to your cluster:
+
+```bash
+subo compute deploy core
+```
+
+### 7. Setup minikube tunneling
+
+Let’s tell minikube to forward requests to port 80 to our cluster!
+
+```bash
+minikube tunnel
+```
+
+### 8. Create an editor token
+
+In order to test our editor, we’re going to come up with a function name, and create a token so we can access it!
+
+This can only be done as [an API call to the control plane service](https://docs.suborbital.dev/compute/integrate-the-function-editor/code-editor) from within your cluster. Since we’re currently not running an app in our cluster, we’ll just make the call from within!
+
+First, we’ll need the name of our control plane pod:
```

Comment:
How could we change this control plane reference to something that means the same thing but doesn't refer to the control plane?

Reply:
Could you please expand on this @LauraLangdon? I understand we're doing away with references to the control plane, so this would be an API call?